docker/perf_layer_env.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+#
+# Enable the performance layers by setting the evironment variables.
+# Run vkcube and make sure it runs without fault when performance layers are 
+# enabled.
+# Test if each layer write to its corresponding output file. 
+
+set -e
+
+# Create a temp output directory for log files. 
```

Comment:
This line is fine as-is, because the comment describes what 'we' do in this place in code, so no need to change it. Same below where I didn't comment.